Update for Pack: Gatewatcher AionIQ (#38289)
demisto/content: Packs/Gatewatcher-AionIQ/ReleaseNotes/1_3_0.md

#### Integrations

##### GCenter

> - Documentation and metadata improvements.

Reviewer: ```suggestion
Documentation and metadata improvements.
```

##### GCenter 103

> - New: This integration now provides commands for interacting with the GCenter.
> - The integration commands provides the following features:

Reviewer: ```suggestion
New: This integration now provides commands for interacting with the GCenter.
The integration commands provides the following features:
```

> 	- Interact with the GCenter API
> 	- User can run commands in the WarRoom
>         - User can create Playbooks from these commands

Reviewer: ```suggestion
- Interact with the GCenter API.
- User can run commands in the War Room.
- User can create Playbooks from these commands.
```

#### Layouts

##### Gatewatcher Alert Incident Layout

> - Updated Layout for displaying Indicators
> 
> ##### Gatewatcher Incident Layout
> 
> - Updated Layout for displaying Indicators

Reviewer: ```suggestion
Updated the layout for displaying indicators.

##### Gatewatcher Incident Layout

Updated the layout for displaying indicators.
```
